From 8d407dc301a02d568fdfaf9d0d16f59e1ad32429 Mon Sep 17 00:00:00 2001
Message-Id: <8d407dc301a02d568fdfaf9d0d16f59e1ad32429.1683756318.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1683756318.git.sharlatanus@gmail.com>
References: <cover.1683756318.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 9 May 2023 08:04:20 +0100
Subject: [PATCH 1/4] gnu: python-can: Update to 4.2.0, fix build.

* gnu/packages/python-xyz.scm (python-can): Update to 4.2.0, fix build.
---
 gnu/packages/python-xyz.scm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/python-xyz.scm b/gnu/packages/python-xyz.scm
index 1417031788..c67dae29a1 100644
--- a/gnu/packages/python-xyz.scm
+++ b/gnu/packages/python-xyz.scm
@@ -2548,14 +2548,14 @@ (define-public python-aenum
 (define-public python-can
   (package
     (name "python-can")
-    (version "4.0.0")
+    (version "4.2.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "python-can" version))
        (sha256
         (base32
-         "0pcbdgdw94lc2sxd56w6wdsshrs9dd5d3wp0x4sfd0drzx32inar"))))
+         "1w5sdzxivpd3pw4pypwnjlksvfimdb93qnlddbrh5f13flhsgg8g"))))
     (build-system python-build-system)
     (arguments
      `(#:phases (modify-phases %standard-phases
-- 
2.39.2

